lib/Image.php

```diff
@@ -42,9 +43,9 @@ public static function create()
     {
         try {
             if (extension_loaded('imagick')) {
-                return new Adapter\Imagick();
+                return Pimcore::getContainer()->get(Adapter\Imagick::class);
```

Comment:
this throws exception when running `./vendor/bin/pimcore-install`
![image](https://user-images.githubusercontent.com/5137917/115017526-f5ccac80-9eb6-11eb-9250-073416d8c80d.png)